### PR TITLE
Refined comments and added function in TenantView

### PR DESCRIPTION
**Pull request by Shahraz98**
## How Has This Been Tested?

- [x] Visually
- [x] Unit
- [ ] Integration

## Types of changes

- [ ] Bug fix (fix)
- [ ] New feature (feat)
- [x] Improvement/refactoring (refactor)
- [x] Code style (style)
- [ ] Build (build):

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

- [x] The PR title is filled in correctly.
- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`main`).
- [x] My code follows the code style of this project.
- [x] I've added relevant changes to the documentation

### CODE REVIEW

File: frontend/src/Helpers.tsx [hunk 1]
```diff
@@ -3,10 +3,10 @@
 // SPDX-FileCopyrightText: 2019 Georg Schwarz <georg. schwarz@fau.de>
 
 /**
- * Helper function to add comma separator if not the last element.
- * @param index the index of current element in array.
- * @param length the length of array.
- * @returns comma or null.
+ * Adds a comma separator if not the last element.
+ * @param {number} index the index of current element in array.
```
Comment (JulianTS) [inline frontend/src/Helpers.tsx:7]: missing `-` after the param names

File: frontend/src/Helpers.tsx [hunk 2]
```diff
@@ -3,10 +3,10 @@
...
+ * @param {number} length the length of array.
```
Comment (JulianTS) [inline frontend/src/Helpers.tsx:8]: missing `-` after the param names

File: frontend/src/components/NavBar.tsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ import { useLocation, useNavigate } from 'react-router-dom'
 import { updateFilterAccordingToNav } from '../store/filterSlice'
 import { Topology } from '../enum'
 
-//Removed 'Message' from this array for now
+// Removed 'Message' from this array for now
```
Comment (JulianTS) [inline frontend/src/components/NavBar.tsx:24]: this comment can be deleted entirely if you want

File: frontend/src/components/custom/CustomFilter.tsx [hunk 4]
```diff
@@ -26,36 +37,33 @@ const CustomFilter: React.FC<CustomFilterProps> = ({ currentView }) => {
...
 
-	const filteredCheckboxes = (searchQuery: string, completeArray: string[]) => {
-		let filteredArr = []
-		filteredArr = completeArray.filter((item: string) => {
-			return item.includes(searchQuery)
-		})
-		return filteredArr
-	}
-
-	const filteredClusters = filteredCheckboxes(
```
Comment (JulianTS) [inline frontend/src/components/custom/CustomFilter.tsx:37]: was this deleted code from line 29-37 deprecated or inefficient? And I hope it's working as intended :D

Comment (Shahraz98) [reply frontend/src/components/custom/CustomFilter.tsx:37]: I renamed the function to filteredBySearchQuery and moved it into the Helpers.tsx file.

**Review by JulianTS — APPROVED** [review]
LGTM

**Pull request merged**